### PR TITLE
Create ansible role for installing agent

### PR DESCRIPTION
**Pull request by sjonpaulbrown**
## Description
This PR creates an Ansible role that will assist in the installation of the Metrika agent on the linux machines. Basic documentation is provided to ensure that the usage is documented.

### CODE REVIEW

File: ansible/roles/metrika-agent-install/templates/agent.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    # disable_fingerprint_validation: disables fingerprint validation on startup.
+    #
+    # Fingerprint validation is enabled by default and the agent will exit
+    # immediately if checksums of the newly retrieved hostname and the cached do
+    # not match. Checksum is cached under /Users/sjonpaul.brown/.cache/metrikad/fingerprint.
+    disable_fingerprint_validation: false
+
+  # metrics_addr: string, network address to get Prometheus metrics about the
+  # agent's runtime.
+  metrics_addr: :9000
```
Comment (kortheof) [inline ansible/roles/metrika-agent-install/templates/agent.yml.j2:61]: This variable should not be hardcoded, but rather configurable by the end-user.

The default value should be: `127.0.0.1:9000`

The reason is that the current hardcoded value of `:9000` starts the /metrics HTTP server of the agent listening on 
`0.0.0.0:9000`.

**Comment by ulfox** [comment]
Stale CI job closed this due to a wrong window. Fixed it and re-opened the PR

**Comment by cosmix** [comment]
@ulfox @hasiotis let's review anew, fix (e.g. what @kortheof mentioned above) and merge in `contrib` as discussed please!

File: ansible/roles/metrika-agent-install/templates/agent.yml.j2 (hunk 1 above)
Comment (iogakos) [reply ansible/roles/metrika-agent-install/templates/agent.yml.j2]: Addressed in a2516967.

**Comment by iogakos** [comment]
> @ulfox @hasiotis let's review anew, fix (e.g. what @kortheof mentioned above) and merge in `contrib` as discussed please!

Addressed both.

**Review by iogakos — APPROVED** [review]

**Pull request merged**